docs/meta.md

```diff
@@ -77,9 +77,9 @@ The following keys are optional:
 
 * `uses`: a map of names and skills
```

Comment:
There's also another entry for uses above which needs conversion. Can't comment there as it's out of the diff.